src/content/docs/nrql/nrql-syntax-clauses-functions.mdx

```diff
@@ -2962,11 +2953,11 @@ Use non-aggregator functions for non-numerical data in NRQL queries.
 
           <tr>
             <td>
-              'valueC[-1:]'
+              `valueC[-1:]
             </td>
 
             <td>
-              Returns the last n list values, where n is a negative number before the colon(i.e [-n:]). In this case the list value at index 2 will be returned.
+              Returns the last _n_ list values, where _n_ is a negative number before the colon(i.e `[-n:]`). In this case the list value at index `2` will be returned.
```

Comment:
```suggestion
              Returns the last **n** list values, where **n** is a negative number before the colon(for example, `[-n:]`). In this case the list value at index `2` will be returned.
```

Reply:
Bold instead of italics is fine with me, but I made the same change of `n` -> `_n_` in another part of the doc as well.

Is this related to the run failing? before the changed I added today this PR was building fine but I see no issues with my changes from this morning

Reply:
I changed both to use `**`